packages/popover/src/Popover.stories.tsx

```diff
@@ -188,25 +195,25 @@ export const ScrollableContainer: StoryFn<PopoverStoryProps> = ({
 }: PopoverStoryProps) => {
   const [active, setActive] = useState<boolean>(false);
   const portalRef = useRef<HTMLElement | null>(null);
-  const portalContainer = useRef<HTMLDivElement | null>(null);
+  const scrollContainer = useRef<HTMLDivElement | null>(null);
 
   const position = referenceElPositions[refButtonPosition];
```

Comment:
Should this be moving the popover when it scrolls out of the container, or have we not enabled that?